src/r_data/models/models_obj.cpp

```diff
@@ -187,32 +187,47 @@ void FOBJModel::ParseFaceSide(const FString &sideStr, OBJFace &face, int sidx)
 	int origIdx;
 	if (sideStr.IndexOf("/") >= 0)
 	{
-		TArray<FString> sides = sideStr.Split("/");
+		TArray<FString> sides = sideStr.Split("/", FString::TOK_KEEPEMPTY);
```

Comment:
I'll have to change each forward slash to something else because having two forward slashes together is valid OBJ syntax, but FScanner recognizes that as a line comment.